src/tools/interop/idt/Dockerfile

```diff
@@ -0,0 +1,61 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+FROM debian:bookworm
+# generic deps
+RUN apt-get update
+RUN apt-get install -y \
+    apt-utils \
+    git \
+    pkg-config \
+    wget
+# python
+RUN apt-get install -y python3.11
+RUN apt-get install -y \
+    python3-pip \
+    python3.11-venv
+# bt deps
+RUN apt-get install -y \
+    glib-2.0 \
+    libglib2.0-dev
+RUN apt-get install -y \
+    libbluetooth-dev \
+    libboost-python-dev \
+    libboost-thread-dev
+# adb
+RUN apt-get install -y \
+    adb \
+    usbutils
+RUN usermod -aG plugdev $(whoami)
+# pcaps / net tools
+RUN apt-get install -y \
```

Comment:
Just fyi, I guess, I think its better for layer reusability if you clump all of the apt-get install commands together in a single RUN step and also finish it with the [recommended](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get) `rm -rf /var/lib/apt/lists/*` to minimize the layer size.

> [To build your own image, you create a Dockerfile with a simple syntax for defining the steps needed to create the image and run it. Each instruction in a Dockerfile creates a layer in the image. When you change the Dockerfile and rebuild the image, only those layers which have changed are rebuilt. This is part of what makes images so lightweight, small, and fast, when compared to other virtualization technologies.](https://docs.docker.com/get-started/overview/#images)

Reply:
When I'm changing things in the image often, I find it helpful to split up apt so you can benefit from layer cache for a portion of the dependencies, but yea this suggestion is of course better outside of that context.

Reply:
Thanks I've written this as suggested now!